go/vt/orchestrator/db/generate_patches.go

```diff
@@ -549,27 +549,27 @@ var generateSQLPatches = []string{
 	`
 		ALTER TABLE
 			database_instance
-			ADD COLUMN semi_sync_master_timeout INT UNSIGNED NOT NULL DEFAULT 0 AFTER semi_sync_master_enabled
+			ADD COLUMN semi_sync_primary_timeout INT UNSIGNED NOT NULL DEFAULT 0 AFTER semi_sync_primary_enabled
 	`,
 	`
 		ALTER TABLE
 			database_instance
-			ADD COLUMN semi_sync_master_wait_for_slave_count INT UNSIGNED NOT NULL DEFAULT 0 AFTER semi_sync_master_timeout
```

Comment:
these variable names map directly to the mysql global variables of same name, which is the reason why the master/slave terminology appears here in the first place. Changing these column names will break that implicit connection (which is fine, just pointing this out).

Reply:
Yes I saw that. I thought we can cleanup the nomenclature from our end, though this would break the one to one mapping between the same names. Should I change it back?

Reply:
this is fine, just pointing out